src/App.js

```diff
@@ -139,6 +143,27 @@ class App extends Component {
   };
 
   render() {
+
+    if ( 
+      this.props.availableOrganisations.length === 0 && 
+      this.props.isFetchingOrganisations === false &&
+      this.props.timesFetchedOrganisations > 0
+    ) {
+      const norolesMessage = this.props.intl.formatMessage({ id: "authorization.no_roles_message", defaultMessage: "Dear user, \nYou seem not to be in any organisations that can acces the management pages. \nTherefore you are redirected to the mainpage" });
```

Comment:
Acces -> access, and a period at the end.

Reply:
done

Reply:
done